src/primo/browzine-primo-adapter.js

```diff
@@ -559,6 +574,12 @@ browzine.primo = (function() {
       pdfIconWidth = "15";
       pdfIconMarginRight = "1.5px";
       articlePDFDownloadLinkText = browzine.articleExpressionOfConcernText || "Expression of Concern";
+    } else if (showProblematicJournalArticleNoticeUI(problematicJournalArticleNoticeUrl)) {
+      directToPDFUrl = problematicJournalArticleNoticeUrl;
+      pdfIcon = "https://assets.thirdiron.com/images/integrations/browzine-retraction-watch-icon.svg";
+      pdfIconWidth = "15";
```

Comment:
I wonder if there's a better way to dry up this code, since for retractions, EOC, and problematic journals, the pdfIconWidth and pdfIconMarginRight are all the same size...

Reply:
Agreed that it would be nice.  I'd like to focus that on a refactor-only PR in the future if you're cool with that, because we're already adding a hunk of new functionality with this PR.